wgsl/index.bs

```diff
@@ -10408,10 +10408,10 @@ An assignment through a [=full reference=] is a [=full assignment=].
 An assignment through a [=partial reference=] is a [=partial assignment=].
 
 When the uniformity rules in subsequent sections refer to the value for a
-function-scope variable used as an RValue, it means the value of the variable
-prior to evaluation of the RValue expression.
+function-scope variable used as an [=RHSValue=], it means the value of the variable
```

Comment:
Not critical but I find myself reading RHSValue as Right-Hand Side Value often. Would it make sense for article to be "a" or is it not appropriate to do?

Reply:
I could go either way with this.

Reply:
I also don't have a strong opinion.

Reply:
Thanks for informing on the potential! Maybe we can think on it in a separate PR or discussion, no need to stall this PR on it!